Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMultiView.css

```diff
@@ -7,81 +7,63 @@ div.tabbedMultiView
   position: relative;
   margin: 0;
   padding: 0;
-  background-color: white;
 }
 
 div.tabbedMultiView > div.wrapper
 {
   position: absolute;
-  background-color: white;
   top: 0;
   bottom: 0;
   left: 0;
   right: 0;
-  width: auto;
-  height: auto;
+  padding-top: var(--remotion-themed-spacing-double);
+  padding-right: var(--remotion-themed-spacing-double);
+  padding-left: var(--remotion-themed-spacing-double);
```

Comment:
```suggestion
```
The padding needs to go into contentborder, otherwise the scrollbar is on the wrong indent-level.